Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -33,10 +33,11 @@ public typealias PurchaseResultData = (transaction: StoreTransaction?,
 /**
  Completion block for ``Purchases/purchase(product:completion:)``
  */
-public typealias PurchaseCompletedBlock = @MainActor @Sendable (StoreTransaction?,
-                                                                CustomerInfo?,
-                                                                PublicError?,
-                                                                Bool) -> Void
+// todo: main actor here
+public typealias PurchaseCompletedBlock = @Sendable (StoreTransaction?,
+                                                     CustomerInfo?,
+                                                     PublicError?,
+                                                     Bool) -> Void
```

Comment:
this is the most concerning one, where we are effectively doing a public-facing change. But it's one that removes restrictions, rather than adding. I expect maybe it adds some Sendable warnings?
Then again... How in the world did we add this in a minor in the first place?